phpunit.xml.dist

```diff
@@ -1,15 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
-         colors="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
-         processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false"
-         bootstrap="vendor/autoload.php">
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    backupGlobals="false"
+    backupStaticAttributes="false"
+    beStrictAboutChangesToGlobalState="true"
+    beStrictAboutOutputDuringTests="true"
+    beStrictAboutTestsThatDoNotTestAnything="true"
```

Comment:
This reveals that https://github.com/php-coveralls/php-coveralls/blob/b86d8d83c92b68e0e5c94f063d833318686f12e9/tests/Bundle/CoverallsBundle/Config/ConfiguratorTest.php#L60-L70 is a risky test:

```
There was 1 risky test:

1) PhpCoveralls\Tests\Bundle\CoverallsBundle\Config\ConfiguratorTest::throwInvalidConfigurationExceptionOnLoadEmptyYmlIfSrcDirNotFound
This test did not perform any assertions
```

Reply:
which phpunit it was introduced?
how having it in config will behave for lower phpunit ?

Reply:
Quick search revealed it's been around for quite a while, see https://github.com/sebastianbergmann/phpunit/issues/914.